login_server/src/cpp/test/main.cpp

```diff
@@ -27,7 +30,7 @@ void fillTests()
 	//	gTests.push_back(new LoginTest());
 }
 
-void runMysql(std::string sqlQuery)
+int runMysql(std::string sqlQuery)
```

Comment:
bool instead of int as return type? You return exactly two values -1 an 0? Or is there a reason you choose int?